src/components/input/index.tsx

```diff
@@ -81,12 +85,13 @@ const Input = forwardRef<HTMLInputElement, Props>(
       min,
       max,
       position,
-      debounce,
+      debounce: delay,
```

Comment:
I know I commented on this before, but seeing how we're just aliasing it, it doesn't make sense in the full context. I'd piggy back from my previous request and name it `debounce` again. the code reads better with it actually

Reply:
I think the alias is fine here because we import the loadash.debounce as `debounce` as well and this would avoid any confusions between the function and the delay time

Reply:
then we could go for naming the `prop` delay as well. that would be a breaking change, but given that it never worked.. I would not care much about that